client/app/routes/index.coffee

```diff
@@ -74,6 +77,9 @@ module.exports = class Router extends Backbone.Router
     component determined by the step param.
     ###
     register: (step = 'preset') -> require.ensure [], =>
+        # Load onboarding stylesheet
+        AppStyles = require '../styles/onboarding.styl'
```

Comment:
As the register process is not handled by this code anymore, I think that this line should be there https://github.com/cozy/cozy-proxy/blob/development/client/app/application.coffee#L67

Maybe we can create a named handler to clarify this part of `Application`. Something like :

```coffeescript
@router.route "#{step.route}", "route:#{step.route}", () =>
    @handleStepRoute(step)

// [...]
handleStepRoute: (step) ->
    # Load onboarding stylesheet
    AppStyles = require './styles/onboarding.styl'

    # actual code
    @layout.showChildView 'content',
        new StepView
            model: new StepModel step: step 
```

What do you think ?